### ISSUE TITLE
Excluding an entire folder

### ISSUE THREAD

**Issue by johnthagen** [kopach/karma-sabarivka-reporter#26]
I've tried to do this:

```js
    coverageReporter: {
      include: [
        // Specify include pattern(s) first
        'src/**/*.(ts|js)',
        // Then specify "do not include" patterns (note `!` sign on the
        //   beginning of each statement)
        '!src/mod/*',
        '!src/**/*.spec.(ts|js)',
        '!src/**/*.module.(ts|js)',
        '!src/**/environment*.(ts|js)',
      ]
    },
```

But source files in `src/mod`, `src/mod/submod1`, etc still shows up in the final coverage report. Is there a prescribed way to ignore an entire folder recursively (including all files and subfolders)?

**Comment by johnthagen** [comment]
Previous to using this plugin, we did something like this in `angular.json`:

```
        "test": {
           ...
            "codeCoverageExclude": ["src/mod"]
          }
```

But it seems like this plugin might interact with that as it no longer seems to work to exclude that folder from the coverage report.

**Comment by kopach** [comment]
Hi @johnthagen. Thank you for your feedback. This is interesting, haven't faced that before. I'll check it. Anyway, note, `karma-sabarivka-reporter` is supposed to only **enrich** your current code coverage. Means, if some file has been added to code coverage report by any other tool – it cannot be excluded by `karma-sabarivka-reporter`.
Btw, do you think it could be useful to make `karma-sabarivka-reporter` to also exclude some files from coverage report?

**Comment by kopach** [comment]
@johnthagen, could you, please, update me with the following info: which exact version of Angular, angular-cli, karma-sabarivka-reporter, npm and node you use?
I need that in order to prepare exact same environment you've described. Thank you.

**Comment by kopach** [comment]
I've just made example angular9 app and everything seems to work as expected. Note, however, that in latest angular9 `angular.json` config is a bit different, from what you've written. 
Instead of 

```
        "test": {
           //...
            "codeCoverageExclude": ["src/mod"]
          }
```

You now need to specify it like so:
```
        "test": {
          "options": {
           //...
            "codeCoverageExclude": ["src/mod/**"]
		  }
         }
```

**Comment by johnthagen** [comment]
@kopach Thanks! The `/**` was what I was missing (I actually did have the `"options": {` but didn't cut and paste properly.

Everything works now in Angular 9. I did still need to keep the `'!src/mod',` part.

I think it would be very useful to new users if there was a short paragraph in the README explaining the interplay between the two exclude methods, as it's very likely other users will need to understand this.

Thanks for the great plugin!
